internal/strategies/gradualrolloutuserid.go

```diff
@@ -0,0 +1,44 @@
+package strategies
```

Comment:
What is the file-name conventions in go, snake-case, camel-case? Feels a bit uncommon to call a file `gradualrolloutuserid.go`.